bpython/line.py

```diff
@@ -6,6 +6,7 @@
 
 from itertools import chain
 from typing import Optional, NamedTuple
+import re
```

Comment:
Nitpick: we usually have `import` first, then `from ... import ...`.

Reply:
Done.